packages/codegen/src/generate-descriptors.ts

```diff
@@ -470,6 +475,46 @@ export function getAssetId(lookup: MetadataLookup) {
   return
 }
 
+const knownSignedExtensions = new Set<string>([
+  // "CheckNonce",
+  // "CheckMortality",
+  // "ChargeTransactionPayment",
+  // "ChargeAssetTxPayment",
+  // "CheckGenesis",
+  // "CheckMetadataHash",
+  // "CheckSpecVersion",
+  // "CheckTxVersion",
+])
```

Comment:
I'm unsure about this, hence why it's commented out. These are the extensions that papi pre-fills if they are not set. But technically, the user could still set them.

Maybe we exclude only the ones that can be filled from the regular API? i.e. ChargeAssetTxPayment, CheckMortality and CheckNonce?

Reply:
I'm open to leaving them, though; it is not a "no-go" for me.